### PR TITLE
Add economy stats command

### PR DESCRIPTION
**Pull request by DeathlyPlays**

### CODE REVIEW

File: spacialgaze-plugins/economy.js [hunk 1]
```diff
@@ -320,4 +320,19 @@ exports.commands = {
...
+	currency: 'economystats',
+	stardust: 'economystats',
+	bucks: 'economystats',
+	economystats: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		const users = Object.keys(Db.currency.object());
+		const total = users.reduce(function (acc, cur) {
+			return acc + Db.currency.get(cur);
+		}, 0);
+		let average = Math.floor(total / users.length) || '0';
```
Comment (HoeenCoder) [inline spacialgaze-plugins/economy.js]: the optional value should be the number 0, not the string 0.

Comment (DeathlyPlays) [reply spacialgaze-plugins/economy.js]: Which zero lol line 332 or 333

File: spacialgaze-plugins/economy.js [hunk 2]
```diff
@@ -320,4 +320,19 @@ exports.commands = {
...
+		let output = "There " + (total > 1 ? "are " : "is ") + total + currencyName(total) + " circulating in the economy. ";
+		output += "The average user has " + average + currencyName(average) + ".";
```
Comment (HoeenCoder) [inline spacialgaze-plugins/economy.js]: currencyName is not a function

File: spacialgaze-plugins/economy.js [hunk 3]
```diff
@@ -320,4 +320,19 @@ exports.commands = {
 		user.updateIdentity();
 		this.sendReply('Your symbol has been removed.');
 	},
+
+	currency: 'economystats',
+	stardust: 'economystats',
+	bucks: 'economystats',
```
Comment (HoeenCoder) [inline spacialgaze-plugins/economy.js]: Id perfer not to do bucks as an alias here

File: spacialgaze-plugins/economy.js (hunk 1 above)
Comment (HoeenCoder) [reply spacialgaze-plugins/economy.js]: the one thats a string `'0'` on 333 the one on 332 is already a number

File: spacialgaze-plugins/economy.js [hunk 4]
```diff
@@ -323,16 +323,15 @@ exports.commands = {
...
 		if (!this.runBroadcast()) return;
 		const users = Object.keys(Db.currency.object());
 		const total = users.reduce(function (acc, cur) {
 			return acc + Db.currency.get(cur);
 		}, 0);
 		let average = Math.floor(total / users.length) || '0';
-		let output = "There " + (total > 1 ? "are " : "is ") + total + currencyName(total) + " circulating in the economy. ";
-		output += "The average user has " + average + currencyName(average) + ".";
+		let output = "There " + (total > 1 ? "are " : "is ") + total + currencyPlural(total) + " circulating in the economy. ";
+		output += "The average user has " + average + currencyPlural(average) + ".";
```
Comment (HoeenCoder) [inline spacialgaze-plugins/economy.js]: currencyPlural isnt a function...

Comment (DeathlyPlays) [reply spacialgaze-plugins/economy.js]: You used that in another command OwO

Comment (HoeenCoder) [reply spacialgaze-plugins/economy.js]: which one

**Pull request merged**